### PR TITLE
[Feature] Speaker Modal Animation

### PR DESCRIPTION
**Pull request by runyasak**
# Scope of work

Implement animation on open `SpeakerModal`.

# Screenshot
Kapture 2020-01-26 at 16 04 11

### CODE REVIEW

File: website2/.vuepress/local-components/SpeakerModal.vue [hunk 1]
```diff
@@ -258,4 +272,24 @@ p {
 .text-center {
   text-align: center;
 }
+
+@keyframes modal-expand {
+  from {
+    width: 150px;
```
Comment (dtinth) [inline website2/.vuepress/local-components/SpeakerModal.vue]: Let’s animate with transform instead.

File: website2/.vuepress/local-components/SpeakerModal.vue [hunk 2]
```diff
@@ -140,9 +142,12 @@ export default {
   position: relative;
   box-sizing: border-box;
   z-index: -2;
+
+  animation-timing-function: linear;
+  animation: modal-expand 1.5s, modal-opacity 1s;
```
Comment (dtinth) [inline website2/.vuepress/local-components/SpeakerModal.vue]: Would be cool if the animation is more snappy. How about 0.4s?

A good animation is one that doesn’t catch too much attention ah.

File: website2/.vuepress/local-components/SpeakerModal.vue (hunk 1 above)
Comment (runyasak) [reply website2/.vuepress/local-components/SpeakerModal.vue]: I have implement with transform that make smooth animated.

File: website2/.vuepress/local-components/SpeakerModal.vue (hunk 2 above)
Comment (runyasak) [reply website2/.vuepress/local-components/SpeakerModal.vue]: I have decreased to 0.4s. But not sure that look cool or not.  😂

**Comment by dtinth** [comment]
On mobile the close button is not working ah x_x

**Comment by runyasak** [comment]
> On mobile the close button is not working ah x_x

I have updated `z-index` to fix it. 😄

**Comment by dtinth** [comment]
It works now, please feel free to merge krubbb

**Pull request merged**